Add 0.6 release announce (#199)
open-cluster-management-io/open-cluster-management-io.github.io: content/en/community/releases.md

Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.5.0`. Continue reading to view upcoming releases:

## `v0.6.0`, on 21st, January 2022

Open Cluster Management team is proud to annouce the release of OCM v0.6.0! We made many enhancements on core components and introduce some new addons.

- [First release of cluster-proxy addon](https://github.com/open-cluster-management-io/cluster-proxy)
- [First release of managed-serviceaccount addon](https://github.com/open-cluster-management-io/managed-serviceaccount)

> - [Syn status of applied resources in ManifestWork](https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/29-manifestwork-status-feedback)

Reviewer: ```suggestion
- [Sync status of applied resources in ManifestWork](https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/29-manifestwork-status-feedback)
```

- [Placement extensible scheduling](https://github.com/open-cluster-management-io/enhancements/blob/main/enhancements/sig-architecture/32-extensiblescheduling)
- [Helm chart interface for addon framework](https://github.com/open-cluster-management-io/addon-framework/pull/62)

> 

Reviewer: ```suggestion
- [Placement API support for multicloud-operators-subscription](https://github.com/open-cluster-management-io/multicloud-operators-subscription/pull/55)

```

> We also add many new functions in [clusteradm](https://github.com/open-cluster-management-io/clusteradm) and enhacements on documentation in the website.

Reviewer: ```suggestion
We also added many new functions in [clusteradm](https://github.com/open-cluster-management-io/clusteradm) and enhancements on documentation in the website.
```

To see details of the changelogs in this release:
- registration v0.6.0 [changelog](https://github.com/open-cluster-management-io/registration/blob/v0.6.0/CHANGELOG/CHANGELOG-v0.6.md)
- work v0.6.0 [changelog](https://github.com/open-cluster-management-io/work/blob/v0.6.0/CHANGELOG/CHANGELOG-v0.6.md)
- placement v0.3.0 [changelog](https://github.com/open-cluster-management-io/placement/blob/v0.3.0/CHANGELOG/CHANGELOG-v0.3.md)
- addon-framework v0.2.0 [changelog](https://github.com/open-cluster-management-io/addon-framework/blob/v0.1.0/CHANGELOG/CHANGELOG-v0.2.md)
- registration-operator v0.6.0 [changelog](https://github.com/open-cluster-management-io/registration-operator/blob/v0.5.0/CHANGELOG/CHANGELOG-v0.6.md)
- multicloud-operators-subscription v0.6.0 [changelog](https://github.com/open-cluster-management-io/multicloud-operators-subscription/blob/v0.6.0/CHANGELOG/CHANGELOG-v0.6.md)

> - cluster-proxy v0.1.3 [changelog]

Reviewer: ```suggestion
- cluster-proxy v0.1.3 [repo](https://github.com/open-cluster-management-io/cluster-proxy)
```

Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.5.0`. Continue reading to view upcoming releases:

## `v0.6.0`, on 21st, January 2022

> Open Cluster Management team is proud to announce the release of OCM v0.6.0! We made many enhancements on core components and introduced some new addons.

Reviewer 2: The Open Cluster Management team ...

- [First release of cluster-proxy addon](https://github.com/open-cluster-management-io/cluster-proxy)
- [First release of managed-serviceaccount addon](https://github.com/open-cluster-management-io/managed-serviceaccount)
- [Sync status of applied resources in ManifestWork](https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/29-manifestwork-status-feedback)
- [Placement extensible scheduling](https://github.com/open-cluster-management-io/enhancements/blob/main/enhancements/sig-architecture/32-extensiblescheduling)
- [Helm chart interface for addon framework](https://github.com/open-cluster-management-io/addon-framework/pull/62)
- [Placement API support for multicloud-operators-subscription](https://github.com/open-cluster-management-io/multicloud-operators-subscription/pull/55)

> We also added many new functions in [clusteradm](https://github.com/open-cluster-management-io/clusteradm) and enhancements on documentation in the website.

Reviewer 2: ... and enhanced the website documentation.

> To see details of the changelogs in this release:

Reviewer 2: See details in the release changelogs:

- registration v0.6.0 [changelog](https://github.com/open-cluster-management-io/registration/blob/v0.6.0/CHANGELOG/CHANGELOG-v0.6.md)
- work v0.6.0 [changelog](https://github.com/open-cluster-management-io/work/blob/v0.6.0/CHANGELOG/CHANGELOG-v0.6.md)
- placement v0.3.0 [changelog](https://github.com/open-cluster-management-io/placement/blob/v0.3.0/CHANGELOG/CHANGELOG-v0.3.md)
- addon-framework v0.2.0 [changelog](https://github.com/open-cluster-management-io/addon-framework/blob/v0.1.0/CHANGELOG/CHANGELOG-v0.2.md)

> - registration-operator v0.6.0 [changelog](https://github.com/open-cluster-management-io/registration-operator/blob/v0.5.0/CHANGELOG/CHANGELOG-v0.6.md)

Reviewer 3: The link is unavailable. should be `https://github.com/open-cluster-management-io/registration-operator/blob/v0.6.0/CHANGELOG/CHANGELOG-v0.6.md`?

Author: thanks for reminding. fixed
